skflow/estimators/dnn.py

```diff
@@ -17,48 +17,18 @@
 
 from sklearn.base import ClassifierMixin, RegressorMixin
 
-from skflow.estimators.base import TensorFlowEstimator
+from skflow.estimators.base import TensorFlowEstimator, ESTIMATOR_COMMON_DOCSTRING
 from skflow import models
+from skflow.util.doc_utils import Appender
 
 
 class TensorFlowDNNClassifier(TensorFlowEstimator, ClassifierMixin):
-    """TensorFlow DNN Classifier model.
```

Comment:
I see. Different libraries have different thoughts on this. I guess this
could prepare generating spinex doc for Skflow later on so users would
browse documentation pages instead of diving into the code. Imagine some
class bodies are long and users don't want to scroll down a lot to find the
documentation for it? Instead, having a indexed documentation page would be
better. Let me know what you think.
On Jan 15, 2016 12:17 AM, "Illia Polosukhin" <notifications@github.com>
wrote:

> In skflow/estimators/dnn.py
> <https://github.com/tensorflow/skflow/pull/80#discussion_r49824075>:
>
> >
> >
> >  class TensorFlowDNNClassifier(TensorFlowEstimator, ClassifierMixin):
> > -    """TensorFlow DNN Classifier model.
>
> I'm a bit torn about this... I understand why you are trying to do this,
> but, for example, sklearn has for all it's the estimators a doc string
> explicitly - like here
> https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/tree/tree.py
> and it kinda make sense when you are just browsing code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/skflow/pull/80/files#r49824075>.
>